src/edit/items/ItemForm.js

```diff
@@ -352,14 +341,17 @@ class ItemForm extends React.Component {
     const effectiveLocation = get(initialValues, ['effectiveLocation', 'name'], '-');
 
     return (
-      <form onSubmit={handleSubmit} data-test-item-page-type={initialValues.id ? 'edit' : 'create'}>
+      <form
+        onSubmit={handleSubmit}
```

Comment:
Please move callback props to the bottom.

Reply:
@VictorSoroka1 just curious, what is the benefit of having callback props at the bottom? should it be included in our codestyle (or already)?

Reply:
For a better scanability of props. I think it is not in our codestyle, but I would prefer so.

Reply:
@VictorSoroka1 

> better scanability
is a subjective mark, don't you think?) Maybe it makes sense to discuss this as a common cod style and as a result make an eslint rule for this or just a note in code style if most of devs are agree. What do you think?

Reply:
I agree that it is a bit subjective. I don't know a kind of process of discussing such things. How do you see it?

Reply:
some people think props in alphabet order are more readable, so yep, it's better to discuss outside and mb update our code style - I think it can be raised in stripes channel

Reply:
right, let's continue in slack https://folio-project.slack.com/archives/C210UCHQ9/p1587461296346600